vipps-recurring-api.md

```diff
@@ -666,6 +667,15 @@ Vipps recurring payment will have a description with follow format
 
 **Note:** `description` cannot be longer than 45 characters.
 
+### Cancel a charge
+
+You can cancel charges that are in the `pending`, `due` or `reserved` state.
+If you cancel a charge that is `PARTIALLY_CAPTURED`, the remaining funds on the charge will be released back to the customer.
+
+**Note:** if you cancel an agreement, there is no need to cancel the charges that belongs to the agreement. This will be done automatically by Vipps.
```

Comment:
```suggestion
**Note:** If you cancel an agreement, there is no need to cancel the charges that belong to the agreement. This will be done automatically by Vipps.
```